Frontend/linkedout/src/main/pages/LoginPage/LoginPage.tsx

```diff
@@ -15,12 +26,16 @@ const LoginPage = () => {
             <div className='login-container'>
                 <TextField
                     className='username-field'
-                    label="Username"
+                    label="Username/Email"
```

Comment:
I would lean towards using email instead of a username since we are already going to have to check the emails when registering to make sure they are unique

Reply:
Sounds good, I'll readjust